### PR TITLE
format code with dprint

### PR DESCRIPTION
**Pull request by g-plane**
Since markup_fmt supports Vento, this PR use dprint to format files.

### CODE REVIEW

File: src/_includes/css/header.css [hunk 1]
```diff
@@ -7,7 +7,7 @@
     margin: 0;
     text-wrap: balance;
 
-    + p {
+    & + p {
```
Comment (oscarotero) [inline src/_includes/css/header.css]: Is there a way to disable this behavior? `&` character is not needed any more for nested rules in CSS.

Comment (g-plane) [reply src/_includes/css/header.css]: Currently no. If we delete `&`, it will produce syntax errors. Let me fix the Malva formatter.

Comment (kwaa) [reply src/_includes/css/header.css]: I think it would be more intuitive to use the `&` symbol in nesting css.

Comment (oscarotero) [reply src/_includes/css/header.css]: It's added only if the nesting selector starts with characters like `+` or `>`. But not if the nesting selector is, for example, `footer { a { ... }}`.

**Review by oscarotero — APPROVED** [review]

**Comment by oscarotero** [comment]
Thank you!!

**Pull request merged**